documentation/src/main/asciidoc/topics/con_protostream.adoc

```diff
@@ -0,0 +1,15 @@
+[id='protostream_{context}']
+= ProtoStream marshalling
+{brandname} includes an implementation of the ProtoStream API that can marshall Java objects as Protocol Buffers (Protobuf).
```

Comment:
There is only one implementation of the ProtoStream API so I would change this to something  like  "includes the Protostream API which allows Java Objects to be marshalled as Protocol Buffers (Protobuf)"